docs/src/examples/components/Dropdown/Types/DropdownExampleSingleSelection.shorthand.steps.js

```diff
@@ -0,0 +1,7 @@
+module.exports = [
+  steps => steps.click('.ui-dropdown .ui-button').snapshot('Shows list'),
```

Comment:
should we use `Dropdown.className` and `Button.className` to avoid regressions when this changes.
Same for other occurances